tests/Umbraco.Tests.Common/TestHelpers/DocumentUpdateHelper.cs

```diff
@@ -17,10 +17,11 @@ public static UpdateDocumentRequestModel CreateInvariantDocumentUpdateRequestMod
             {
                 Segment = null,
                 Culture = null,
-                Name = createModel.InvariantName!,
+                Name = createModel.Variants.FirstOrDefault(v => v.Culture is null && v.Segment is null)?.Name
```

Comment:
There are a few places now where we get the `InvariantName` from `Variants.FirstOrDefault(v => v.Culture is null && v.Segment is null)?.Name`.  Do you think it's worth reintroducing this property as a calculated property or method, e.g. `GetInvariantName()`?

Reply:
At this point I don't see an urgent need. We could consider an extension method to that end later on, or maybe an extension method to get the "invariant variant".